BackEnd.java

```diff
@@ -117,8 +119,7 @@ private void playerMoveOnField() {
         }
 
 	if (beatsPossible) {
-	    // Only figures able to beat another figure should be
```

Comment:
I added a hard line break in this comment to keep the line width under 72 characters.

Sticking with that number might seem odd (it is from a time where programmers used punch cards), but it really simplifies reading code when I have two windows side by side as the code fits perfectly on that limited amount of space.